### ISSUE TITLE
Is this gem maintained?

### ISSUE THREAD

**Issue by bibstha** [code-mancers/invoicing#52]
Looking for some invoicing gems and this one looks like it has good documentation.
But the last commit is in 2015. So curious if there are any forks maintained or do you guys plan to support this still?

**Comment by iffyuva** [comment]
@bibstha no time unfortunately. If you have proper use case, let me know. We can support it.

**Comment by bibstha** [comment]
Does it at least run on Rails5? I like the concepts mentioned in the documentation and would love to see it atleast compatible with Rails 5.1.

**Comment by iffyuva** [comment]
We can land this support @bibstha . Give us this weekend.

**Comment by weber-s** [comment]
Hello,

I am reviving this issue. Apparently there is almost everything ready for rails5 with the PR #83 and the branch of @thegeorgeous  https://github.com/code-mancers/invoicing/tree/update-ruby-rails-versions .

I am relying on this gem for a non-profit Internet Service provider invoicing logic, and we need to move on rails5 now. Invoicing is the only gem that limit the upgrade right now. I really think this gem is a very nice one and having a release on rubygem that support rails 5.2 would be really great!

Any news on it?

**Comment by etcook** [comment]
We're actively using it on Rails 6, if that means anything to you.

**Comment by etcook** [comment]
@weber-s I have a fork where I've compiled some network changes into one which allow for usage on rails >= 5. I also patched an issue with frozen string literals that will allow you to use it with Ruby >= 2.6. It needs to be tested some more, but it's operational in a production environment. Admittedly we're only using a fraction of the capabilities, though, so not sure as to coverage.

**Comment by iffyuva** [comment]
@etcook can you please open a PR, I will merge it and do a release?

**Comment by etcook** [comment]
@iffyuva Actually, I just attempted to do a PR and it looks like after a merge with your master, the two are actually now in parity. So we're effectively using your master in production right now.

While I have your attention, I'd like to do some functional PRs, especially around support for UUIDs and automatically recalculating totals, etc. Will these be accepted?

**Comment by iffyuva** [comment]
@etcook yes, definitely. Please open them so that we can discuss and merge those PRs.

**Comment by weber-s** [comment]
Hello,

Is a release on rubygem planned ? I strongly prefer using gem from here than github.
